sympy/stats/tests/test_continuous_rv.py

```diff
@@ -951,6 +951,19 @@ def test_rayleigh():
     assert cdf(X)(x) == 1 - exp(-x**2/(2*sigma**2))
     assert diff(cdf(X)(x), x) == density(X)(x)
 
+def test_reciprocal():
+    a = Symbol("a", real=True)
+    b = Symbol("b", real=True)
+
+    X = Reciprocal('x', a, b)
+    assert density(X)(x) == Piecewise((0, a > x), (1/(x*(-log(a) + log(b))), b >= x), (1, True))
+    assert cdf(X)(x) == (log(x) - log(a)) / (log(b) - log(a))
+    X = Reciprocal('x', 5, 30)
+
+    assert E(X) == 25/(log(30) - log(5))
+    assert P(X < 4) == S.Zero
+    assert P(X < 20) == log(20) / (log(30) - log(5)) - log(5) / (log(30) - log(5))
+    assert cdf(X)(10) == (log(10) - log(5)) / (log(30) - log(5))
```

Comment:
Please add tests for checking if the correct assumptions are raised.
Use `raises` for writing such tests.

Reply:
@czgdp1807  i have made the changes. Please take a look.